input/pagecontent/StructureDefinition-ncpi-sample-intro.md

```diff
@@ -11,7 +10,7 @@ The NCPI Sample profile is based on the standard resource type, [Specimen](https
 
 #### Recommended Practices
 
-* The NCPI Sample profile is broken up into "NCPI Collected Sample" and "NCPI Non-collected Sample"; be mindful of selecting the right kind of sample to represent your data
+* The NCPI Sample profile includes collection information. If this data is unavailable, you should set collection method to "unknown".
```

Comment:
We likely want to expand this further- though I don't know if we must do it immediately.